packages/api/src/types/rpc.ts

```diff
@@ -11,13 +11,15 @@ import { ApiTypes, PromiseResult, Push, RxResult, UnsubscribePromise } from './b
 export interface RpcRxResult<F extends AnyFunction> extends RxResult<F> {
   json (...args: Parameters<F>): Observable<Json>;
   raw (...args: Parameters<F>): Observable<Raw>;
+  def: DefinitionRpc;
```

Comment:
Just one small comment - would adjust this to be `meta`, just to align with what we have in the rest for this type of info.

Reply:
Thanks Jaco. Change has been made, can you please check again.